README.md

```diff
@@ -68,14 +68,21 @@ string endpointUrl = "<endpointUrl>";
 sfdc37:SalesforceConfiguration salesforceConfig = {
     baseUrl: endpointUrl,
     clientConfig: {
-        auth: {
-            scheme: http:OAUTH2,
-            accessToken: accessToken,
-            refreshToken: refreshToken,
-            clientId: clientId,
-            clientSecret: clientSecret,
-            refreshUrl: refreshUrl
-        }
+       auth: {
+           scheme: http:OAUTH2,
+           config: {
+               grantType: http:DIRECT_TOKEN,
+               config: {
+                   accessToken: accessToken,
+                   refreshConfig: {
+                       refreshUrl: refreshUrl,
```

Comment:
Please fix refreshUrl to use a public constant from the module wherever applicable.

Reply:
Salsforce refreshUrl we are getting from the user, since that will differ from user to user

Reply:
Got it!